hal/src/rtl872x/usbd_driver.cpp

```diff
@@ -263,8 +263,10 @@ int RtlUsbDriver::setupReply(SetupRequest* r, const uint8_t* data, size_t size)
             // some common cases of returning not a lot of data generated on stack
             memcpy(tempBuffer_, data, size);
             CHECK_RTL_USB_TO_SYSTEM(usbd_ep0_transmit(rtlDev_, tempBuffer_, size));
+            HAL_Delay_Milliseconds(2);
```

Comment:
The build is currently failing because the bootloader doesnt include `HAL_Delay_Milliseconds()`. If we are ok without this change in the bootloader (I think we are?) I think we need to wrap this with 
```
#if MODULE_FUNCTION != MOD_FUNC_BOOTLOADER
...
#endif
```

Reply:
Yeah, `HAL_Delay_Milliseconds()` is based on FreeRTOS, actually `HAL_Delay_Microseconds()` also works for me, but let's use your method to keep the bootloader as-is.